nucliadb/nucliadb/writer/api/v1/services.py

```diff
@@ -90,6 +99,101 @@ async def set_entities(request: Request, kbid: str, group: str, item: EntitiesGr
         )
 
 
+@api.post(
+    f"/{KB_PREFIX}/{{kbid}}/entitiesgroups",
+    status_code=200,
+    name="Create Knowledge Box Entities Group",
+    tags=["Knowledge Box Services"],
+    openapi_extra={"x-operation_order": 1},
+)
+@requires(NucliaDBRoles.WRITER)
+@version(1)
+async def create_entities_group(
+    request: Request, kbid: str, item: CreateEntitiesGroupPayload
+):
+    ingest = get_ingest()
+
+    pbrequest: NewEntitiesGroupRequest = NewEntitiesGroupRequest()
+    pbrequest.kb.uuid = kbid
+    pbrequest.group = item.group
+    pbrequest.entities.custom = True
+    if item.title:
+        pbrequest.entities.title = item.title
+    if item.color:
+        pbrequest.entities.color = item.color
+
+    for key, entity in item.entities.items():
+        entitypb = pbrequest.entities.entities[key]
+        entitypb.value = entity.value
+        entitypb.merged = entity.merged
+        entitypb.deleted = False
+        entitypb.represents.extend(entity.represents)
+
+    set_info_on_span({"nuclia.kbid": kbid})
+
+    status: NewEntitiesGroupResponse = await ingest.NewEntitiesGroup(pbrequest)  # type: ignore
+    if status.status == NewEntitiesGroupResponse.Status.OK:
+        return
+    elif status.status == NewEntitiesGroupResponse.Status.KB_NOT_FOUND:
+        raise HTTPException(status_code=404, detail="Knowledge Box does not exist")
```

Comment:
See other comment on raise http exceptions.

I think this will also product sentry errors for us.

Reply:
Same as above